src/pages/wilczy-polmaraton.astro

```diff
@@ -198,15 +205,15 @@ const ytPreviewLink = 'https://www.youtube.com/embed/ftF3hJ-Qfcc';
           <span class="text-center">Kliknij ikonę warstw w lewym górnym rogu mapy, aby sterować widokiem trasy.</span>
         </p>
       </div>
-    </div>
+    </div>  -->
   </section>
-  
+
     <!-- Photos Section -->
     <section class="max-w-7xl mx-auto">
-      <h2 class="text-3xl font-bold text-center">Zdjęcia z 2024</h2>
-      <PhotoGallery photos={marathonPhotos2024} author="Andrzej Jaźwiński" />
+      <h2 class="text-3xl font-bold text-center">Zdjęcia z 2025</h2>
+      <PhotoGallery photos={marathonPhotos2024} author="Paweł Bernas" />
       <div class="text-center mt-4 text-lg">
-        <p>Pełna galeria zdjęć Andrzeja znajduje się <a href="https://drive.google.com/drive/folders/1An5Tm9Pt9qMlrAQ8AxwfY7cEBWW0xlSE?usp=drive_link" class="text-blue-600 hover:underline">tutaj</a>. Zobaczcie również zdjęcia Adama Wrożyny <a href="https://photos.app.goo.gl/Gkmh9iEZHKa7Cm2x8" class="text-blue-600 hover:underline">tutaj</a>.    
+        <p>Pełna galeria zdjęć Pawła znajduje się <a href="https://www.instagram.com/pawelbernas.fotografia" class="text-blue-600 hover:underline">tutaj</a>.    
```

Comment:
here it was suppose to be a link to google drive with photos

instagram was suppose to be a link under "Autor - Paweł Bernas" test, basically `Paweł Bernas` should be a hyperlink.

you just need to extend PhotoGallery component to accept `link` as another optional new prop, and if provided, you need to make author test a hyperlink